test/models/driver_test.rb

```diff
@@ -60,22 +60,68 @@
 
   # Tests for methods you create should go here
   describe "custom methods" do
+    before do
+      @new_driver = Driver.create(name: "Mike", vin: "123", available: true)
+      @new_passenger = Passenger.create(name: "Mike", phone_num: "111-111-1211")
+
+    end
+    let(:trip_1) {
+      Trip.create(driver_id: @new_driver.id, passenger_id: @new_passenger.id, date: Date.today, rating: 5, cost: 1000)
+    }
+    let(:trip_2) {
+      Trip.create(driver_id: @new_driver.id, passenger_id: @new_passenger.id, date: Date.today, rating: 3, cost: 6334)
+    }
+    let(:trip_3){
+      Trip.create(driver_id: @new_driver.id, passenger_id: @new_passenger.id, date: Date.today, rating: 3, cost: 1234)
+    }
     describe "average rating" do
-      # Your code here
+      it "returns average rating" do
+        trip_1
+        trip_2
+        trip_3
+        expect(@new_driver.average_rating).must_be_close_to 3.7
+      end
+
+      it "returns 0 if there is no trip" do
+        expect(@new_driver.average_rating).must_equal 0
     end
+  end
+
+  describe "total_revenue" do
+    it "returns total_revenue" do
+      trip_1
```

Comment:
Same as the conversation -- these tests read strangely to me, since `trip_1` doesn't communicate to me what's going on. It may make more sense to have the `Trip.create()` line in this test